lib/dealership.rb

```diff
@@ -0,0 +1,68 @@
+# require './lib/car'
+#
+#
+#
+class Dealership
+
+  attr_reader :name, :address, :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+    @total_val = 0
+    @inv_by_cost = []
```

Comment:
We don't need to initialize these variables here. It makes more sense that there is a method `total_value` and that it calculates that given the inventory at the time that the method is called.